sys/netbsd/sys.txt

```diff
@@ -340,7 +362,7 @@ unlinkat_flags = 0, AT_REMOVEDIR
 flock_op = LOCK_SH, LOCK_EX, LOCK_UN, LOCK_NB
 faccessat_flags = AT_EACCESS, AT_SYMLINK_NOFOLLOW
 rusage_who = RUSAGE_SELF, RUSAGE_CHILDREN
-rlimit_type = RLIMIT_AS, RLIMIT_CORE, RLIMIT_CPU, RLIMIT_DATA, RLIMIT_FSIZE, RLIMIT_MEMLOCK, RLIMIT_NOFILE, RLIMIT_NPROC, RLIMIT_RSS, RLIMIT_STACK, RLIMIT_NTHR, RLIMIT_STACK
+rlimit_type = RLIMIT_AS, RLIMIT_CORE, RLIMIT_CPU, RLIMIT_DATA, RLIMIT_FSIZE, RLIMIT_MEMLOCK, RLIMIT_NOFILE, RLIMIT_NPROC, RLIMIT_RSS, RLIMIT_STACK, RLIMIT_NTHR, RLIMIT_SBSIZE
```

Comment:
Can we sort these entries according to their numbers?

https://nxr.netbsd.org/xref/src/sys/sys/resource.h#88

Why to remove `RLIMIT_STACK`?

Reply:
Yes, would add it.

Reply:
Please sort, so RLIMIT_CPU, RLIMIT_FSIZE, etc.